assets/stylesheets/_fix-image-paths.scss

```diff
@@ -4,16 +4,19 @@ input[type='checkbox'] {
 		content: url( '../../wp-calypso/public/images/checkbox-icons/checkmark-primary.svg' );
 	}
 }
-$payment-cc-brands: 'amex', 'diners', 'discover', 'jcb', 'mastercard', 'unionpay', 'visa';
-$payment-types: 'alipay', 'bancontact', 'giropay', 'eps', 'ideal', 'paypal', 'p24', 'brazil-tef',
-'wechat', 'sofort';
-@each $brand in $payment-cc-brands {
-	&.is-#{$brand} {
-		background-image: url( '../../wp-calypso/public/images/upgrades/cc-#{$brand}.svg' );
+
+.payment-logo {
```

Comment:
How come we are adding another class?

Reply:
I think this fixes the "mastercard/visa card images not showing" problem.

Reply:
To fix the visa mastercard images. Basically it never worked the way it was.